Uplift.Api/Controllers/CategoryController.cs

```diff
@@ -15,18 +15,72 @@ public CategoryController(ICategoryService categoryService)
             _categoryService = categoryService;
         }
 
-        [HttpPost("create-category")]
-        public IActionResult Create([FromBody] Category model)
+        // GET: api/Category
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Category>>> GetCategories()
```

Comment:
For now, it is fine to use Category here, Just I wanna mention in the future we will define ViewModels for each entity and use VMs here.

Reply:
Thank you. I wrote it down.